tests/smc/test_inner_kernel_tuning.py

```diff
@@ -281,7 +281,7 @@ def test_with_adaptive_tempered(self):
 
         def parameter_update(state, info):
             return extend_params(
```

Comment:
calling extend params shouldn't be needed anymore

Reply:
I'm still using extend_params since it needs to convert to an array and add a leading dimension of length one.  I could have added a separate helper but it'd do the same thing as extend_params but just without the jnp.repeat part

Reply:
I think it is worth to modify `extend_params` into something simple without the `jnp.repeat`.

Would this work?
```python
def extend_params(params: ArrayLikeTree):
    return jax.tree.map(lambda a: jnp.expand_dims(a, 0), params)
```

Reply:
That would work, perhaps with potentially a `jnp.asarray()` included.  The concern I have with modifying `extend_params` is I'd assume there's still a use case for it with initializing unshared parameters?  

A current use case is for the remaining tests that are using duplicated parameters for testing the unshared case.

Reply:
@ciguaran Is there any other use case besides dimension matching initially?

Reply:
There's none.

Reply:
I'd assume there's still a use case for it with initializing unshared parameters -> that is a good concern although I think that in most cases we wouldn't initialize by "extending" with copies. We would do it by sampling from some probability distribution. So you can modify it as Junpeng suggested.

Reply:
ah, that makes sense.  thanks